docs/pages/versions/unversioned/introduction/testing-with-jest.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Testing with Jest
+---
+
+Adding Jest to your Expo app has gotten pretty simple, to help with this process, we've even created a [Jest preset](https://github.com/expo/expo/tree/master/packages/jest-expo) for you. Let's get started.
+
+## Installation
+
+First we need to install jest-expo: `yarn add jest-expo --dev` **or** `npm i jest-expo --save-dev`
```

Comment:
The first thing we'll want to do is install `jest-expo`, it's a Jest preset that mocks out the native side of the Expo SDK and handles some configuration for you. To install `jest-expo` as a development dependency, run `npm install jest-expo --save-dev`.